.github/workflows/publish.yml

```diff
@@ -233,8 +246,9 @@ jobs:
         with:
           apt: ${{ inputs.libraries }}
       - id: build
-        uses: OpenAstronomy/build-python-dist@bbb0e1c5b132893999ea56d77bd4b526e0097c7d  # v1.0.1
+        uses: OpenAstronomy/build-python-dist@9a4b2bd6bda5c48ef3c12b53504ee2263ed032e3 # ???
```

Comment:
note to reviewers: I suppose this other repo should receive a new version tag

Reply:
Good idea, I've now tagged [`v1.1.0`](https://github.com/OpenAstronomy/build-python-dist/releases/tag/v1.1.0)

Reply:
awesome, edited !